src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexFindOptimizations.cs

```diff
@@ -83,6 +84,7 @@ public RegexFindOptimizations(RegexNode root, RegexOptions options)
             bool dfa = (options & RegexOptions.NonBacktracking) != 0;
             bool compiled = (options & RegexOptions.Compiled) != 0 && !dfa; // for now, we never generate code for NonBacktracking, so treat it as non-compiled
             bool interpreter = !compiled && !dfa;
+            bool usesRfoTryFind = !compiled;
```

Comment:
Is the name a typo?

Reply:
No, it's referring to **R**egex**F**ind**O**ptimizations though obviously not doing a good job of making the connection :)